week3/src/components/LevelButton.jsx

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable react/prop-types */
+import styled from "styled-components";
+
+function LevelButton({ setCards, setDifficulty, getMixedCardList }) {
+  const handleEasy = () => {
+    setCards(getMixedCardList(5));
+    setDifficulty("easy");
+  };
+
+  const handleNormal = () => {
+    setCards(getMixedCardList(7));
+    setDifficulty("normal");
+  };
+
+  const handleHard = () => {
+    setCards(getMixedCardList(9));
+    setDifficulty("hard");
+  };
+
+  return (
+    <>
+      <ButtonWrapper>
+        <EasyBtn onClick={handleEasy}>Easy</EasyBtn>
+        <NormalBtn onClick={handleNormal}>Normal</NormalBtn>
+        <HardBtn onClick={handleHard}>Hard</HardBtn>
+      </ButtonWrapper>
+    </>
+  );
+}
+
+// button
+const ButtonWrapper = styled.div`
+  display: flex;
+  gap: 2vw;
+  justify-content: center;
+  align-items: center;
+  margin: 15vh 0 8vh 0;
+`;
+const EasyBtn = styled.button`
+  width: 10vw;
+  height: 4vh;
+  background-color: ${({ theme }) => theme.colors.lightYellow};
+  border: none;
+  border-radius: 30px;
+  &:hover {
+    background-color: ${({ theme }) => theme.colors.skyBlue};
+  }
+`;
```

Comment:
p5: 선택된 버튼 색상을 변경하려면, 현재 난이도를 상태로 받아와서 버튼의 난이도와 일치하는지 여부를 판단하면 어떨까요?
지은님 코드 기준으로 예시를 드리자면 


```suggestion
// prop으로 difficulty 상태를 받아왔다고 가정
  return (
    <>
      <ButtonWrapper>
        <EasyBtn onClick={handleEasy} $isSelected={ "easy" === difficulty }>Easy</EasyBtn>
        <NormalBtn onClick={handleNormal} $isSelected={ "normal" === difficulty }>Normal</NormalBtn>
        <HardBtn onClick={handleHard} $isSelected={ "hard" === difficulty }>Hard</HardBtn>
      </ButtonWrapper>
    </>
  );
}

// button
const ButtonWrapper = styled.div`
  display: flex;
  gap: 2vw;
  justify-content: center;
  align-items: center;
  margin: 15vh 0 8vh 0;
`;
const EasyBtn = styled.button`
  width: 10vw;
  height: 4vh;
  background-color: ${({ $isSelected, theme }) =>
    $isSelected ? theme.colors.selectedColor : theme.colors.lightYellow};
  border: none;
  border-radius: 30px;
  &:hover {
    background-color: ${({ theme }) => theme.colors.skyBlue};
  }
`
```

Reply:
현재 상태를 가져와서 조건부 스타일링을 하면 쉽게 구현할 수 있네요!
이걸 몰라서 헤맸네요 ..
감사합니다 ☺️